CHANGES.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+### 1.81 - 2021-05-01
+
+##### Deprecated :hourglass_flowing_sand:
+
+- `loadCRN` was deprecated and will be removed in Cesium 1.82. It will be replaced with support for KTX2.[9478](https://github.com/CesiumGS/cesium/pull/9478)
+- `loadKTX` was deprecated and will be removed in Cesium 1.82. It will be replaced with support for KTX2. [9478](https://github.com/CesiumGS/cesium/pull/9478)
```

Comment:
```suggestion
- `loadKTX` has been deprecated and will be removed in CesiumJS 1.82. It will be replaced with support for KTX2. [#9478](https://github.com/CesiumGS/cesium/pull/9478)
```